build.sh

```diff
@@ -53,7 +53,7 @@ KERNEL_IMAGE="uImage-${BUILDCONFIG}"
 DEBIAN_DIR="${BUILD_DIR}/debian"
 BOOT_FILE_OUTPUT_DIR="${DEBIAN_DIR}/boot"
 
-INITRAMFS_MODULES_REQUIRED="loop.ko"
+INITRAMFS_MODULES_REQUIRED="loop.ko sn65dsi84.ko usb251xb.ko"
```

Comment:
There used to be more modules here for the imx6, do we know that we don't need them anymore?

Reply:
I don't know but the listed drivers do not list any dependencies. I cannot test the screen, but we know usb does not work yet.

Reply:
I think this needs more investigation, I tested USB on my board just now and it is working. However, I still have USB as compiled in I guess, because I don't see any drivers in lsmod. We might need more or other modules to be enabled. 
In dmesg i can see the following: 

root@ultimakersystem-0013953a482e:~# dmesg | grep usb
[    0.442136] root@ultimakersystem-0013953a482e:~# dmesg | grep usb
[    0.442136] usbcore: registered new interface driver usbfs
[    0.447414] usbcore: registered new interface driver hub
[    0.452701] usbcore: registered new device driver usb
[    0.457988] **usb_phy_generic usbphynop1**: usbphynop1 supply vcc not found, using dummy regulator
[    0.466547] usb_phy_generic usbphynop2: usbphynop2 supply vcc not found, using dummy regulator
[    1.596224] usbcore: registered new interface driver usb-storage
[    1.602322] usbcore: registered new interface driver usb_ehset_test
[    1.609713] **usbmisc_imx** 32e40200.usbmisc: 32e40200.usbmisc supply vbus-wakeup not found, using dummy regulator
[    1.619964] usbmisc_imx 32e50200.usbmisc: 32e50200.usbmisc supply vbus-wakeup not found, using dummy regulator
[    2.057767] usb 2-1: new high-speed USB device number 2 using ci_hdrc
[    2.521777] usb 2-1.4: new high-speed USB device number 3 using ci_hdrc
[    3.255949] usbcore: registered new interface driver usbhid
[    3.261550] **usbhid**: USB HID core driver
[ 3083.996479] usb 2-1.4.4: new high-speed USB device number 4 using ci_hdrc
[ 3084.116019] usb-storage 2-1.4.4:1.0: USB Mass Storage device detected
[ 3084.125232] scsi host0: usb-storage 2-1.4.4:1.0

Reply:
we probably also need imxdrm